tools/cassandra/cqlclient.go

```diff
@@ -112,12 +112,16 @@ const (
 var _ schema.SchemaClient = (*CqlClientImpl)(nil)
 
 // NewCQLClient returns a new instance of CQLClient
-func NewCQLClient(cfg *CQLClientConfig) (CqlClient, error) {
+func NewCQLClient(cfg *CQLClientConfig, expectedConsistency gocql.Consistency) (CqlClient, error) {
 	var err error
 
 	cqlClient := new(CqlClientImpl)
 	cqlClient.cfg = cfg
 	cqlClient.nReplicas = cfg.NumReplicas
+	consistency := gocql.All
+	if expectedConsistency == gocql.Quorum {
```

Comment:
I'm not sure you need this? I think you can probably just pass in expectedConsistency as a config parameter?

Reply:
didn't want to tamper with the struct but yeah can do.